src/components/EditEquipmentListingGeneralPanel/EditEquipmentListingGeneralPanel.js

```diff
@@ -66,6 +74,8 @@ const EditEquipmentListingGeneralPanel = props => {
               isEquipment: true,
             },
           };
+          if (freePlan.length)
+            updateValues.publicData.freePlan = freePlan;
```

Comment:
I think there is a compile error right here, because the `updateValues` doesn't contain `publicData` property.